articles/building-an-appointment-scheduling-telegram-bot/index.md

```diff
@@ -0,0 +1,514 @@
+### What is Fauna?
+
+Fauna is a client-side serverless document database that uses GraphQL and the Fauna Query Language (FQL) to support various data types and relational databases in a serverless API. You can learn more about Fauna in their official documentation [here](https://docs.fauna.com/fauna/current/).
+
+### Creating A Fauna Database
+
+To make use of Fauna, you have to signup on their website [here](https://dashboard.fauna.com/accounts/register). After signing up, you can create a new database by clicking on the `CREATE DATABASE` button on your dashboard. Provide a name for your database and save it.
+
+![database_dashboard](/engineering-education/building-an-appointment-scheduling-telegram-bot/database_dashboard.png)
+
+#### Creating the Fauna Collections
+
+For this article, you will need to create two collections in your database. A collection is similar to tables in your conventional database system. The first collection you need to create is the `Users` collection. This is where you will save all the user’s information. The second collection you need to create is the `Appointments` collection. This is where you will create all the user’s appointments. 
+
+To create a collection, click on `CREATE COLLECTION` then provide information for the required fields. You will be required to enter the name of the collection, the `History Days` and `TTL`. The `History Days` field refers to the number of days you want Fauna to retain a historical record of any data in that collection. The `TTL` refers to an expiry date for data in the collection. For example, if the `TTL` is 3, Fauna will automatically delete any data stored in the database three (3) days after the last date you modified it.
+
+![create_collection](/engineering-education/building-an-appointment-scheduling-telegram-bot/create_collection.png)
+
+After saving the collections you just created, you will notice there are no documents in your collections. A document is similar to rows of data in a table as in your conventional database system.
+
+#### Creating a Fauna Index
+
+You will need to create a Fauna index that will allow you to scroll through the data created in your database. To do this, go to the `DB Overview` tab on the left side of your screen, then click on the `New Index` button.
+
+For this article, you will be creating three indexes; `users_index`, `appointment_index`, and `appointment_today_index`. The `users_index` index will enable you to scroll through data in the `Users` collection using the `id` field as a parameter to match. The `appointment_index` index will enable you to scroll through data in the `Appointments` collection using the `user_id` field as a parameter to match. The `appointment_today_index` index will allow you to scroll through data in the `Appointments` collection. It will use the `user_id` and `date_due` fields as parameters to match.
+
+![create_index](/engineering-education/building-an-appointment-scheduling-telegram-bot/create_index.png)
+
```

Comment:
You are saying "for this article" a lot. Please change it. For example:
We will create three indexes...
and why there is a semicolon after "three indexes"?